src/controller.cpp

```diff
@@ -591,6 +591,10 @@ int Controller::run(const CliArgsParser& args)
 				std::cout << strprintf::fmt(_("%" PRIu64 " unreachable feeds found. See "
 							"`cleanup-on-quit` in newsboat(1) for details."), num_feeds)
 					<< std::endl;
+				if (cfg.get_configvalue("cleanup-on-quit") == "nudge") {
+					std::cout << _("Press Enter to continue") << std::endl;
+					std::cin.get();
```

Comment:
This requires pressing enter.
It might be nicer to allow any keypress but haven't found a nice way yet (probably requires setting some unbuffering option on stdin, either manually or using something like ncurses)

Reply:
Done using ncurses